src/static/css/style.css

```diff
@@ -550,6 +550,7 @@ table .td--right {
     padding: 10px 20px 20px 20px;
     z-index: 1000;
     background-color: #fff;
+    width: 40vw;
```

Comment:
![image](https://github.com/user-attachments/assets/d832926a-0a9b-48ab-b698-64c4b7a526f3)
Chez moi c'est trop large et ça dépasse de la carte, est ce que ce n'est pas mieux en % :
```suggestion
    width: 90%;
```?